src/components/dataCode/components/Datacode.tsx

```diff
@@ -1,20 +1,74 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useTimer } from 'use-timer';
 import {
   DataCodeWrapper,
   SingleDataCodeWrapper,
   SingleDataCode,
+  Footer,
+  FooterContent,
+  RefreshButton,
 } from '../styles/datacode.s';
-const code = [7, 2, 1, 9, 3, 4];
 
 export const Datacode = () => {
+  const [code, setCode] = useState([0, 0, 0, 0, 0, 0]);
+  const [codeExpired, setCodeExpired] = useState(0);
+  const [refresh, setRefresh] = useState(0);
+  //useTimer library
+  const { time, start, reset, status } = useTimer({
+    initialTime: 5 * 60,
+    endTime: 0,
+    timerType: 'DECREMENTAL',
+  });
+
+  const getNewCode = async () => {
```

Comment:
Fix newCode function to handle an exception : too much requests (e.g. Block requests when timer state 240 ~ 300)